eclair-core/src/test/scala/fr/acinq/eclair/TestDatabases.scala

```diff
@@ -95,13 +98,13 @@ object TestDatabases {
       val dbs = Databases.SqliteDatabases(connection, connection, connection, jdbcUrlFile_opt = Some(jdbcUrlFile))
       dbs.copy(channels = new SqliteChannelsDbWithValidation(dbs.channels))
     }
-    override def close(): Unit = ()
+    override def close(): Unit = connection.close()
     // @formatter:on
   }
 
   case class TestPgDatabases() extends TestDatabases {
-    private val pg = EmbeddedPostgres.start()
-    val datasource: DataSource = pg.getPostgresDatabase
+
+    val datasource: DataSource = getNewDatabase
```

Comment:
nit:
```suggestion
    val datasource: DataSource = getNewDatabase()
```

If we want to get rid of the warning that this looks like a getter (which is a dumb warning that only looks at the fact that the function name starts with `get`) we can rename that `createNewDatabase`